src/afrolabs/components/kafka/json_serdes.clj

```diff
@@ -2,10 +2,20 @@
   (:gen-class
    :implements [org.apache.kafka.common.serialization.Serializer]
    :main false)
-  (:require [clojure.data.json :as json]
-            [taoensso.timbre :as log]
-            [clojure.string :as str])
-  (:import [org.apache.kafka.common.header Headers]))
+  (:require
+   ;; [clojure.data.json :as json]
+   [charred.api :as json]
+   [taoensso.timbre :as log]
+   [clojure.string :as str])
```

Comment:
Using the `charred.api` library for loading JSON, which has better performance than `clojure.data.json` and with the same API.